snuba/admin/static/tracing/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Client from "../api_client";
+import QueriesDisplay from "../components/query_display/query_display";
+import { QueryResult } from "../components/query_display/types";
+
+function TracingQueries(props: { api: Client }) {
+  function tablePopulator(queryResult: QueryResult) {
+    var elements = {};
+    if (queryResult.error === undefined) {
```

Comment:
The proper way to check for undefined in javascript is:
`typeof(query.result.error) === "undefined"`
Or if you only need to know whether it's falsy can just be the easier:
`if (queryResult.error)`